cmd/juju/controller/addmodel.go

```diff
@@ -322,14 +321,7 @@ func (c *addModelCommand) Run(ctx *cmd.Context) error {
 	// "Added '<model>' model [on <cloud>/<region>] [with credential '<credential>'] for user '<user namePart>'"
 	ctx.Infof(messageFormat, messageArgs...)
 
-	if _, ok := attrs[config.AuthorizedKeysKey]; !ok {
-		// It is not an error to have no authorized-keys when adding a
-		// model, though this should never happen since we generate
-		// juju-specific SSH keys.
-		ctx.Infof(`
-No SSH authorized-keys were found. You must use "juju add-ssh-key"
-before "juju ssh", "juju scp", or "juju debug-hooks" will work.`)
-	}
+	ctx.Infof("To use \"juju ssh\", \"juju scp\" and \"juju debug-hooks\" ssh public keys need to be added to the model with \"juju add-ssh-key\"")
```

Comment:
I think the message is better, but I think the start is a bit off. Yet I don't have anything that comes to mind.

@tmihoc any clues, you're really good at this :point_up:

Reply:
@SimonRichardson What about the start is bothering you?

I don't know the context for this. (1) Are we anticipating a question a user in this context will actually have or (2) is this a random by-the-way? Without knowing that, all I can say is that 

(a) I don't like the "need to be added" -- I'd rephrase as "make sure to add".

(b) I don't like the "with \"juju add-ssh-key\"" bit -- it suggests you have to do it that way, and that's not true because you can also do this with `terraform-provider-juju` and `python-libjuju`, plus that info is really not-at-issue. I'd make it parenthetical:

> ... add your ssh public keys to the model (e.g., with `juju add-ssh-key`)

or, pretending like the other clients don't exist but still weakening the implications of the non-parenthetical "with":

> ... add your ssh public keys to the model (`juju add-ssh-key`)